docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -1,15 +1,15 @@
 {% set short_lang = 'python' %}
 {% if language == 'JavaScript' %}
     {% set short_lang = 'js' %}
-{% elif language == 'Java' %}
-    {% set short_lang = 'java' %}
+{% elif language == 'Go' %}
```

Comment:
There are still some Java chunks in this file.  In the source file, they're at line 87, line 1163, and line 1207.